website/docs/getting-started/installation.md

```diff
@@ -2,3 +2,74 @@
 title: Installation
 sidebar_position: 4
 ---
+
+odo can be used as a CLI tool and as an IDE plugin; it can be run on Linux, Windows and Mac systems.
+
+## CLI Binary installation
+odo supports amd64 architecture for Linux, Mac and Windows.
+Additionally, it also supports amd64, arm64, s390x, and ppc64le architectures for Linux.
+
+See the [release page](https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/) for more information.
+
+### Installing odo on Linux/Mac
+```shell
+OS="$(uname | tr '[:upper:]' '[:lower:]')"
+ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')"
+curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-$OS-$ARCH -o odo
+sudo install odo /usr/local/bin/
+```
+
+### Installing odo on Windows
+1. Download the [odo-windows-amd64.exe](https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe) file.
+2. Rename the downloaded file to odo.exe and move it to a folder of choice, for example C:\odo
+3. Add the location of odo.exe to `%PATH%` variable (refer to the steps below).
+
+#### Setting the PATH variable in Windows 10
+1. Click **Search** and type `env` or `environment`.
+2. Select **Edit environment variables for your account**.
+3. Select **Path** from the **Variable** section and click **Edit**.
+4. Click **New**, add the location where you copied the odo binary (e.g. `C:\odo` in [Step 2 of Installation](#installing-odo-on-windows) into the field or click **Browse** and select the directory, and click **OK**.
+
+#### Setting the PATH variable in Windows 7/8
+1. Click **Start** and in the Search box types `Advance System Settings`.
+2. Select **Advanced systems settings** and click the **Environment Variables** button at the bottom.
+3. Select the **Path** variable from the **System variable** section and click **Edit**.
```

Comment:
Is it really "System variable"? I don't know, so I'm asking. But I think it might be "System Variables".

Reply:
I am not sure. I copy-pasted this from the old website.

Reply:
It is `System variables`. See the 4th image here  https://www.groovypost.com/howto/set-path-system-variable-windows-7-8/.